lighthouse-core/gather/gatherers/iframe-elements.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Gatherer = require('./gatherer.js');
+const pageFunctions = require('../../lib/page-functions.js');
+
+/* eslint-env browser, node */
+
+/**
+ * @return {LH.Artifacts['IFrameElements']}
+ */
+function collectIFrameElements() {
+  // @ts-ignore - put into scope via stringification
+  const iFrameElements = getElementsInDocument('iframe'); // eslint-disable-line no-undef
+  return iFrameElements.map(/** @param {HTMLIFrameElement} node */ (node) => {
+    // @ts-ignore
+    const clientRect = getClientRect(node).toJSON();
+    // Marking 1x1 as non-visible to ignore tracking pixels.
+    const isVisible = (clientRect.width > 1 && clientRect.height > 1);
+    return {
+      id: node.id,
+      src: node.src,
+      clientRect,
+      isVisible,
+      // @ts-ignore
+      isFixed: isVisible && isFixed(node),
+    };
+  });
+}
+
+/** @inheritdoc */
```

Comment:
TIL 😃 

what effects does this end up having for us? at least in vscode I seem to get the same docstring without it, but curious what else takes it into account

Reply:
TBH this was part of the gatherer when I started on the project and it was just never evaluated. Removed for consistency.